### PR TITLE
Refactor /accounts endpoint to use swagger runner interface - Closes #920

### LINKED ISSUE #920: Refactor /accounts to use swagger runner interface
Parent: #899

### PR DESCRIPTION
**Pull request by nazarhussain**
Closes: #920 
Relates: #674

### CODE REVIEW

File: test/functional/http/get/accounts.js [hunk 1]
```diff
@@ -2,319 +2,268 @@
...
-					node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(0);
+				return accountsEndpoint.makeRequest({publicKey: account.publicKey, address: node.gAccount.address}, 200).then(function (res) {
+					res.body.data.should.have.length(0);
+				});
+			});
+		});
+
+		describe.only('secondPublicKey', function () {
+
+			var spkAcccount = node.randomAccount();
```
Comment (Isabello) [inline test/functional/http/get/accounts.js]: i dont like the way this variable is shortened. Can you expand it to be the full name for readability?

File: test/functional/http/get/accounts.js [hunk 2]
```diff
@@ -2,319 +2,268 @@
...
-
-					it('using sort = username:asc should sort accounts by balance in ascending order', function () {
-						return getAccountsPromise('limit=' + 5 + '&sort=username:asc').then(function (res) {
-							node.expect(res).to.have.property('status').to.equal(200);
-							node.expect(res).to.have.nested.property('body.accounts').that.is.an('array').to.have.length(5);
-							node.expect(_.map(res.body.accounts, 'delegate.username')).eql(_.map(_.sortBy(res.body.accounts, 'username'), 'delegate.username'));
-						});
-					});
+			it('using offset = -1 should return error', function () {
+				return accountsEndpoint.makeRequest({offset:  -1}, 400).then(function (res) {
```
Comment (Isabello) [inline test/functional/http/get/accounts.js]: double spaces in some of these objects. Is there a reason for it?

Comment (karmacoma) [reply test/functional/http/get/accounts.js]: Done.

File: test/functional/http/get/accounts.js (hunk 1 above)
Comment (karmacoma) [reply test/functional/http/get/accounts.js]: Done.

**Comment by karmacoma** [comment]
@Isabello Your requested changes have been made, please review.

**Pull request merged**